mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,404 @@
+"""Conversion tool from Neuroscan CNT to FIF
+"""
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+from os import path
+import datetime
+import calendar
+
+import numpy as np
+
+from ...utils import warn, verbose
+from ..constants import FIFF
+from ..utils import _mult_cal_one
+from ..meas_info import _empty_info
+from ..base import _BaseRaw
+from ..utils import _read_str
+
+
+def read_raw_cnt(input_fname, eog=(), ecg=(), emg=(), misc=(),
+                 read_blocks=True, preload=False, verbose=None):
+    """Read CNT data as raw object,
+
+    Note: Channels that are not assigned with keywords ``eog``, ``ecg``,
```

Comment:
I think you can add a note like this: http://www.sphinx-doc.org/en/stable/markup/para.html